Documentation/Columns/Properties/DisplayCond.rst

```diff
@@ -126,6 +126,20 @@ Multiple conditions can be combined::
    ],
 
 
+An example with multiple values and :code:`OR`::
+
+   $GLOBALS['TCA']['tx_mask_table']['columns']['tx_mask_field']['displayCond']['OR'] = [
+      'FIELD:tx_mask_otherfield:=:1',
+      'FIELD:tx_mask_otherfield:=:2',
+      'FIELD:tx_mask_otherfield:=:4',
+   ];
+
+
+This is the same as::
+
+    $GLOBALS['TCA']['tx_mask_table']['columns']['tx_mask_field']['displayCond'] = 'FIELD:tx_mask_otherfield:IN:1,2,4'; 
```

Comment:
```suggestion
        $GLOBALS['TCA']['tx_mask_table']['columns']['tx_mask_field']['displayCond']
		  = 'FIELD:tx_mask_otherfield:IN:1,2,4'; 
```
Try to keep lines in code examples shorter then 80 chars if possible. This prevents horizonal scrolling